docs/deployment-process/operations-runbooks/index.md

```diff
@@ -57,14 +57,20 @@ You can **Run** a **Snapshot** as many times as you want to.
 
 ## Environments Selection
 
-We don't believe that channels or lifecycles (progression) make sense for runbooks. Runbooks can be run on any environments you have access to. The interface has been designed to let you run a runbook quickly, so there's a single `Run on...` screen where you choose environments and specify any `Advanced` options, and then you run it.
+We don't believe that channels or lifecycles (progression) make sense for runbooks. Runbooks can be run on any environments you have access to. The interface has been designed to let you run a runbook quickly, so there's a single `Run...` screen where you choose environments and specify any `Advanced` options, and then you run it.
 
 ## Permissions
 
 Two permissions are available for the top-level Runbooks document: `RunbookView` (for viewing) and `RunbookEdit` (for creating, modifying, and deleting), so if you want to lock down the ability for your teams to create runbooks, you just need to disable these permissions.
 
 Once you create a runbook snapshot and run it, the typical `Deployment` and `Release` permissions are required.
 
+## Publishing
+
+Publishing allows a runbook author (users with `RunbookEdit` permissions) to nominate a Runbook Snapshot as being 'ready to run', informing others that the runbook process (at the given snapshot) has been tried and tested.
```

Comment:
Is it worth say "tried and tested and is an approved version of the runbook." just to really drive it home :)